allennlp/training/trainer.py

```diff
@@ -166,6 +167,16 @@ def train(self) -> None:
                 metrics["loss"] = float(train_loss / batch_num)
                 description = self._description_from_metrics(metrics)
                 train_generator_tqdm.set_description(description)
+
+                batch_num_tot = num_training_batches * epoch + batch_num
+                if self._serialization_prefix and batch_num_tot % self._summary_interval == 0:
```

Comment:
This name changed to `self._serialization_dir`.

Reply:
Ah yes, didn't catch that in the final merge and didn't rerun as I had something running.